src/scripts/main.js

```diff
@@ -1,3 +1,29 @@
 'use strict';
 
-// write your code here
+const spans = document.querySelectorAll('.population');
```

Comment:
```suggestion
const populationList = document.querySelectorAll('.population');
```
consider more meaningful naming